sample_rules/osbs.yaml

```diff
@@ -0,0 +1,282 @@
+           Version:
+             valueRegex: /([\w]+.*)./
+             message: "Label 'Version' is missing or has invalid format"
+             level: "error"
+             required: true
+             reference_url:
+               - "http://docs.projectatomic.io/container-best-practices/#"
+               - "_recommended_labels_for_your_project"
+           Release:
+             valueRegex: /([\w]+.*)./
+             message: "Label 'Release' is missing or has invalid format"
+             level: "error"
+             required: true
+             reference_url:
+               - "http://docs.projectatomic.io/container-best-practices/#"
+               - "_recommended_labels_for_your_project"
+           Architecture:
+             valueRegex: /[\w]*[6,8][4,6]|[.]*86[.]*64/
+             message: "Label 'Architure' is missing or has invalid format: x86, i386, x86_64"
+             level: "error"
+             required: true
+             reference_url:
+               - "http://docs.projectatomic.io/container-best-practices/#"
+               - "_recommended_labels_for_your_project"
+           Url:
+             valueRegex: /([\w]+).+/
+             message: "Label 'Url' is missing or has invalid format"
+             level: "warn"
+             required: true
+             reference_url:
+               - "http://docs.projectatomic.io/container-best-practices/#"
+               - "_recommended_labels_for_your_project"
+           Help:
+             valueRegex: /([\w]+).+/
+             message: "Label 'Help' is missing or has invalid format"
+             level: "warn"
+             required: true
+             reference_url:
+               - "http://docs.projectatomic.io/container-best-practices/#"
+               - "_recommended_labels_for_your_project"
+    FROM: 
+      paramSyntaxRegex: /^[a-z0-9./-]+(:[a-z0-9.]+)?$/
+      rules: 
+        - 
+          label: "is_latest_tag"
+          regex: /latest/
+          level: "error"
+          message: "base image uses 'latest' tag"
+          description: "using the 'latest' tag may cause unpredictable builds. It is recommended that a specific tag is used in the FROM line or *-released which is the latest supported release."
+          reference_url: 
+            - "https://docs.docker.com/reference/builder/"
+            - "#from"
+          label: "no_tag"
+          regex: /^[:]/
+          level: "error"
+          message: "No tag is used"
+          description: "lorem ipsum tar"
+          reference_url: 
+            - "https://docs.docker.com/reference/builder/"
+            - "#from"
+          label: "specified_registry"
+          regex: "/[/]/"
+          level: "warn"
+          message: "using a specified registry in the FROM line"
+          description: "using a specified registry may supply invalid or unexpected base images"
+          reference_url:
+            - "https://docs.docker.com/reference/builder/"
+            - "#entrypoint"
```

Comment:
The special FROM value koji/image-build (with or without a tag suffix) should be allowed as that is how base images can be built.

@lcarva - do we have documentation for that yet?